.gitignore

```diff
@@ -31,3 +31,7 @@ go.work.sum
 
 # editor specific config
 .vscode
+
+ops/bin
+
+.staging
```

Comment:
I'm confused by this. If this directory is gitignored, how does the user add their intent.toml and state.json file to it and push it as part of a pr? Or is that not the intended user flow for generating the report?